azext_iot/_help.py

```diff
@@ -633,6 +633,61 @@
 """
 '''
 
+helps[
```

Comment:
Right above this copy of the `iot hub state` help, there is the original commented out version. Lets remove the duplicate.

![image](https://user-images.githubusercontent.com/28658112/183777356-e07e5ce8-ae62-4f27-820e-ef3e6657c4d4.png)